source/MRViewerApp/MRViewerApp.vcxproj

```diff
@@ -127,7 +127,10 @@
     </Link>
     <PostBuildEvent>
       <Command>copy $(SolutionDir)pluginLibraryList.json $(TargetDir)pluginLibraryList.json
-</Command>
+		if not exist $(TargetDir)meshlib mkdir $(TargetDir)meshlib
```

Comment:
on the one hand, it would be better to add this post build command to each py lib .vcxproj, on the other hand - I don't want to duplicate the code